hack/verify-shellcheck.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set -o errexit
```

Comment:
I think we'd rather leave this enabled, write our shellcheck command to not err, and allow other commands to fail so we don't need `|| exit` sprinkled in places

Reply:
I tested removing this, and I found that shellcheck did not run properly and it just errored out without showing warnings/corrections. I tried this both with and without `|| exit` sprinkled in. I think, perhaps, that shellcheck needs this option unset to continue to log all errors etc. to stdout.

After these tests, I would recommend leaving it commented out.